pkgs/build-support/build-dotnet-module/default.nix

```diff
@@ -30,6 +30,8 @@
 # The NuGet dependency file. This locks all NuGet dependency versions, as otherwise they cannot be deterministically fetched.
 # This can be generated by running the `passthru.fetch-deps` script.
 , nugetDeps ? null
+# The package's NuGet.config file, which contains external package repositories used. This is used in the fetch-deps script described above.
+, nugetConfig ? null
```

Comment:
Would be nice if we could abstract this somehow, and generate the config file ourselves based on the URLs. Not sure how feasible that is though

Reply:
The URLs are in the source, though -- I don't really get what you mean.

Reply:
Oops, i misinterpreted this a bit. You can ignore that comment :sweat_smile: